src/main/angle_pid.c

```diff
@@ -17,18 +17,21 @@ static float apidoutput[ANGLE_PID_SIZE];
 static float lasterror[ANGLE_PID_SIZE];
 
 float angle_pid(int x) {
+  float inverse_angle_error = 1 - fabsf(state.angleerror[x]);
+  float abs_angle_error = fabsf(state.angleerror[x]);
```

Comment:
this helps with cpu load as your not recalculating this over and over, however naming is up for changing.

Reply:
make them `const` then :)

Reply:
> make them `const` then :)

Done :)